src/code/components/animated-sprite.js

```diff
@@ -22,17 +23,31 @@ const FVAnimatedSprite = React.createClass({
     return {"backgroundPositionX": 0};
   },
 
-  componentWillMount() {
+  animate() {
     this.tweenState("backgroundPositionX", {
       easing: tweenState.easingTypes.linear,
       duration: this.props.duration,
       endValue: this.props.frames,
       onEnd: this.props.onEnd
     });
+    this.setState({ started: true });
+  },
+
+  componentDidMount() {
+    if (!this.props.waitForTrigger)
+      this.animate();
+  },
+
+  componentWillReceiveProps(nextProps) {
+    if (!this.state.started && !nextProps.waitForTrigger)
```

Comment:
The current code guarantees that the animation can only be triggered once, which was the behavior I was aiming for. Yours allows the client to trigger multiple animations by repeatedly toggling `waitForTrigger`. After thinking about it a bit, your change could lead to a means to support looped sprite animations, so I'll make the change.